hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/TestTrash.java

```diff
@@ -579,6 +579,7 @@ public void testExistingFileTrash() throws IOException {
     conf.setClass("fs.file.impl", TestLFS.class, FileSystem.class);
     FileSystem fs = FileSystem.getLocal(conf);
     conf.set("fs.defaultFS", fs.getUri().toString());
+    conf.setBoolean(FS_TRASH_CLEAN_TRASHROOT_ENABLE_KEY, true);
```

Comment:
I do not think we need to update this configuration item here. (L582,L639,L649, etc)